src/averageAges.js

```diff
@@ -14,12 +18,25 @@
  *
  * @return {number}
  */
+
+function averageOfArray(array) {
```

Comment:
Try to add more meaningful names and function names should start with a verb.
```suggestion
function getAverageAge (ages) { 
```
If you transfer the repeated part of the logic and transfer the entire array here, then the argument can be called "people".